jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -362,10 +362,11 @@ export class SortingView extends DOMWidgetView {
     if (!plugin) return <div>Plugin not found: {pluginName}</div>
 
     const baseSha1Url = `/sha1`
-    const hither = initializeHitherInterface(msg => {
+    const hither = initializeHitherInterface(baseSha1Url)
+    hither._registerSendMessage(msg => {
```

Comment:
Slight modification was needed in the jupyter extension as to how the hither interface is initialized.